### DISCUSSION TITLE
Shaders not working

### DISCUSSION THREAD

**Discussion by ghost** [mpv-player/mpv#14044] [Q&A]
2024-4-9 - mpv version
Android 13

Need answers 

Shaders like adaptive sharpen not working in Android or in my case .
Does all shaders do not work in Android or any of those not working.
If It is working then please guide me

I have tried on reddit no reply has come. 
If I don't get my answer then this community is also like dead or done atleast in my case.
Thank you 😘

**Comment by kasper93** [comment]
If you reported it in proper place https://github.com/mpv-android/mpv-android and followed the issue template you would know that you have to use `adb logcat -s 'mpv:*' '*:F'` to collect logs from Android and give us that so we can answer your question.

> I have tried on reddit no reply has come.

Speaking of reddit if you spent 2s on typing your question in search engine of your choice, you would find the answer here https://old.reddit.com/r/mpv/comments/15ql1uq/can_i_use_shaders_in_mpv_android/

I'm moving it to discussions as there is nothing actionable to work on here.

**Reply by ghost** [reply]
The reddit post you are talking about is one I have checked before and I wouldn't have bothered going to reddit the first time if you had mentioned the answer.

**Reply by kasper93** [reply]
So, does it work or not? If not you have to get logs and see why.

**Reply by ghost** [reply]
It's night time, I can't use the computer, I won't be able to provide it right now. In the morning I can do 
It did not work as I mentioned

**Reply by ghost** [reply]
please don't feel bad

**Reply by ghost** [reply]
.

**Comment by CounterPillow** [comment]
To heighten the entertainment for the internet peanut gallery, OP also shat up mpv's offtopic IRC channel yesterday:

```
--> Guest31 (~Guest86@171.79.23.61) has joined #mpv-offtopic
<Guest31> Hello does anyone help
<CounterPillow> no
<Guest31> Please guide me how to add shader in mpv Android
<CounterPillow> I refuse
<Guest31> Please
<CounterPillow> No chance
<Guest31> How can anyone in community not help 😞 it's sad that I used trusted on open source
<CounterPillow> Tragic!
<-- Guest31 (~Guest86@171.79.23.61) has left #mpv-offtopic
--> Qqqx (~Qqqx@171.79.23.61) has joined #mpv-offtopic
<Qqqx> H
<-- Qqqx (~Qqqx@171.79.23.61) has quit (Remote host closed the connection)
--> Guest42 (~Guest87@171.79.23.61) has joined #mpv-offtopic
<Guest42> .
<Guest42> .
<Guest42> .
<-- Guest42 (~Guest87@171.79.23.61) has quit (Client Quit)
```

**Reply by ghost** [reply]
You didn't answer me and you just went away And came here for entertainment just wow

**Reply by anatasiajp** [reply]
> You didn't answer me and you just went away And came here for entertainment just wow

Just ignore him lamo, at least if it's someone not an ass he should have point you to help channel, but an asshole is ass.
